.github/workflows/software-environments.yml

```diff
@@ -42,4 +42,21 @@ jobs:
           export PYTHON_VERSION_FORMATTED=$(echo "${{ matrix.python-version }}" | sed 's/\.//g' )
           export COILED_SOFTWARE_ENV_NAME=coiled-runtime-$RUNTIME_VERSION_FORMATTED-py$PYTHON_VERSION_FORMATTED
 
-          coiled env create --name coiled/$COILED_SOFTWARE_ENV_NAME --conda environments/$COILED_SOFTWARE_ENV_NAME.yml
\ No newline at end of file
+          coiled env create --name coiled/$COILED_SOFTWARE_ENV_NAME --conda environments/$COILED_SOFTWARE_ENV_NAME.yml
```

Comment:
Can we move this step to after we've pushed the corresponding docker image and then update the Coiled software environment creation step to use `--container` instead of `--conda`? I believe that was the ultimate motivating factor behind https://github.com/coiled/coiled-runtime/issues/142

Reply:
Testing this now on as PR, commenting out the `test.yaml` workflow. Will uncomment tests after seeing if software build from container works.